cache/remotecache/export.go

```diff
@@ -37,24 +37,131 @@ type Config struct {
 	Compression compression.Config
 }
 
+type CacheType int
+
 const (
 	// ExportResponseManifestDesc is a key for the map returned from Exporter.Finalize.
 	// The map value is a JSON string of an OCI desciptor of a manifest.
 	ExporterResponseManifestDesc = "cache.manifest"
+	CacheManifestSchemaVersion   = 2
```

Comment:
This doesn't need to be public.

If you combine the schemaVersion to a single definition, then this doesn't need to be defined as constant at all.